examples/ingress_create.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Creates deployment, service, and ingress objects. The ingress allows external
+network access within the cluster.
```

Comment:
External network access from the cluster? External and within mean opposite things.

Reply:
Fixed.